joark/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/joark/dokument/PdfDokument.kt

```diff
@@ -152,12 +156,15 @@ class PdfDokument(val dokument: InntektsmeldingDokument) {
             is NyStilling -> addNyStilling(endringsårsak)
             is NyStillingsprosent -> addNyStillingsprosent(endringsårsak)
             is Bonus -> addBonus(endringsårsak)
-            else -> {}
+            is Sykefravaer -> addSykefravaer(endringsårsak)
+            is Nyansatt -> addNyAnsatt()
+            is Feilregistrert -> addFeilregistrert()
+            else -> throw IllegalArgumentException("Mangler hjelpefunksjon for type: ${endringsårsak?.typpe}")
```

Comment:
Er mangelen på hjelpefunksjon problemet?

Reply:
Ja, det er et tegn på at man har lagt til en ny type, uten å lage noen ny håndtering (hjelpemetode) her.

Reply:
Kanskje flisespikking, men da ville jeg heller hatt en feilmelding ala `"Type ${endringsårsak?.typpe} håndteres ikke. Legg til hjelpefunksjon for å løse feilen."`.